src/import.ts

```diff
@@ -27,6 +28,19 @@ export async function importVCard(file: File): Promise<Contact> {
       case "X-CALLSIGN":
         contact.callsign = value;
         break;
+      case "X-STATION-TYPE":
+        contact.cardType = value == "repeater" ? "repeater" : "person";
```

Comment:
should probably at least warn to the console on invalid value

(but also, coercing undefined to "person" is good)